dxr/vcs.py

```diff
@@ -113,8 +114,9 @@ def find_previous_revisions(self, client):
         """
         last_change = {}
         for line in client.rawcommand(['previous-revisions']).splitlines():
-            path, node = line.split(':')
-            last_change[path] = node
+            if ':' in line:
+                path, node = line.split(':', 1)
+                last_change[path] = node
```

Comment:
Good point on adding tests! I've learned that hg does not allow tracking files containing line breaks, so that first check isn't necessary. However I'm having trouble with paths containing a colon since currently I use that as the marker between the two parts.